bindings/go/go.mod

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
It's discussed on general@incubator.a.o that go.mod is like Maven's pom.xml and deserves a license header.

Anyway, it's added in the new version on request (https://github.com/korandoru/hawkeye/commit/d5c0bcfcf96ced5fee7a00949893add762e4787c) 🤣.

Reply:
`mod files must use // comments` :sob:

Reply:
OK. Let me fix it.

Reply:
Fixed now.